app/views/layouts/pdf.html.erb

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>PDFs - Ruby on Rails</title>
+    <%= wicked_pdf_stylesheet_link_tag "npc_pdf" %>
+</head>
+<body>
+    <%= yield %>
+</body>
+</html>
```

Comment:
From the PDF Generator example repo.

I don't believe this is being used anywhere, but I wanted to leave it in case we wanted to utilize it in the future